spec/fixtures/files/wonderschool_necc_onboarding_data.csv

```diff
@@ -1,6 +1,6 @@
-Full Name,Client ID,Provider Name,Date of birth (required),Enrolled in School (Kindergarten or later),Case number,Authorized full day units,Authorized hourly units,Effective on,Expires on,Special Needs Rate?,Special Needs Daily Rate,Special Needs Hourly Rate,Provider Email,Business Zip Code,Business County,Business License,Business QRIS rating,Accredited,Approval #1 - Family Fee,Approval #1 - Begin Date,Approval #1 - End Date,Approval #1 - Allocated Family Fee,Approval #2 - Family Fee,Approval #2 - Begin Date,Approval #2 - End Date,Approval #2 - Allocated Family Fee,Approval #3 - Family Fee,Approval #3 - Begin Date,Approval #3 - End Date,Approval #3 - Allocated Family Fee,Approval #4 - Family Fee,Approval #4 - Begin Date,Approval #4 - End Date,Approval #4 - Allocated Family Fee,Approval #5 - Family Fee,Approval #5 - Begin Date,Approval #5 - End Date,Approval #5 - Allocated Family Fee
-Thomas Eddleman,14047907,Rebecca's Childcare,2010-09-01,No,14635435,276,1656,2020-09-01,2021-08-31,No,,,rebecca@rebecca.com,68845,Corke,Family Child Care Home II,Step 4,Yes,0.00,2020-09-01,2021-08-31,0.00,,,,,,,,,,,,,,,,
-Jacob Ford,31610139,Rebecca's Childcare,2017-09-01,No,14635435,276,1656,2020-09-01,2021-08-31,No,,,rebecca@rebecca.com,68845,Corke,Family Child Care Home II,Step 4,Yes,0.00,2020-09-01,2021-08-31,0.00,,,,,,,,,,,,,,,,
-Linda Rogers,15228275,Rebecca's Childcare,2019-02-21,No,14635435,276,1656,2020-09-01,2021-08-31,No,,,rebecca@rebecca.com,68845,Corke,Family Child Care Home II,Step 4,Yes,0.00,2020-09-01,2021-08-31,0.00,,,,,,,,,,,,,,,,
-Paz Korman,81185504,Rebecca's Childcare,2017-05-17,No,12312445,138,828,2021-01-29,2021-07-31,No,,,rebecca@rebecca.com,68845,Corke,Family Child Care Home II,Step 4,Yes,108.00,2021-01-29,2021-07-31,108.00,,,,,,,,,,,,,,,,
-Becky Falzone,69370816,Kate's Kids,2013-12-26,No,56582912,330,1760,2020-11-24,2021-11-23,Yes,90.77,9.43,kate@kate.com,68845,Corke,Family Child Care Home I,Step 5,No,60.00,2020-11-24,2021-05-23,60.00,85.00,2021-05-24,2021-11-23,85.00,,,,,,,,,,,,
\ No newline at end of file
+Full Name,Client ID,Provider Name,Date of birth (required),Enrolled in School (Kindergarten or later),Case number,Authorized full day units,Authorized hourly units,Authorized weekly hours,Effective on,Expires on,Special Needs Rate?,Special Needs Daily Rate,Special Needs Hourly Rate,Provider Email,Business Zip Code,Business County,Business License,Business QRIS rating,Accredited,Approval #1 - Family Fee,Approval #1 - Begin Date,Approval #1 - End Date,Approval #1 - Allocated Family Fee,Approval #2 - Family Fee,Approval #2 - Begin Date,Approval #2 - End Date,Approval #2 - Allocated Family Fee,Approval #3 - Family Fee,Approval #3 - Begin Date,Approval #3 - End Date,Approval #3 - Allocated Family Fee,Approval #4 - Family Fee,Approval #4 - Begin Date,Approval #4 - End Date,Approval #4 - Allocated Family Fee,Approval #5 - Family Fee,Approval #5 - Begin Date,Approval #5 - End Date,Approval #5 - Allocated Family Fee
```

Comment:
@JonErvin @csprayregen the changes to the CSV onboarding stuff will go into effect when this is merged, but it's non-breaking, you can keep onboarding kids as-is until you add the "Authorized weekly hours" column to the CSV